test/Npgsql.Tests/CommandTests.cs

```diff
@@ -1013,17 +1013,38 @@ public void Statements()
                     Assert.That(cmd.Statements[0].InputParameters[0].Value, Is.EqualTo("foo"));
                     Assert.That(cmd.Statements[0].StatementType, Is.EqualTo(StatementType.Insert));
                     Assert.That(cmd.Statements[0].Rows, Is.EqualTo(1));
-                    Assert.That(cmd.Statements[0].OID, Is.Not.EqualTo(0));
                     Assert.That(cmd.Statements[1].SQL, Is.EqualTo("UPDATE data SET name='b' WHERE name=$1"));
                     Assert.That(cmd.Statements[1].InputParameters[0].ParameterName, Is.EqualTo("p2"));
                     Assert.That(cmd.Statements[1].InputParameters[0].Value, Is.EqualTo("bar"));
                     Assert.That(cmd.Statements[1].StatementType, Is.EqualTo(StatementType.Update));
                     Assert.That(cmd.Statements[1].Rows, Is.EqualTo(0));
-                    Assert.That(cmd.Statements[1].OID, Is.EqualTo(0));
                 }
             }
         }
 
+
+        [Test]
+        public void StatementsOIDs()
```

Comment:
Let's rename to StatementOID (it checks the property OID on type NpgsqlStatement).